src/algebra/invertible.lean

```diff
@@ -161,6 +162,14 @@ def invertible_inv {a : α} [invertible a] : invertible (a⁻¹) :=
 
 end group_with_zero
 
+section ring_char
+
+def invertible_of_ring_char_not_dvd {R : Type*} [field R] {t : ℕ} (not_dvd : ¬(ring_char R ∣ t)) :
```

Comment:
Would you mind also adding the `char_p` predicate version?

Reply:
Done.